reinstall-cli/src/prompt.rs

```diff
@@ -0,0 +1,78 @@
+use crate::users::{get_all_users_keys, UserKeys};
+use anyhow::{ensure, Context, Result};
+
+fn prompt_single_user(user: &crate::users::UserKeys) -> Result<Vec<&crate::users::UserKeys>> {
+    let prompt = format!(
+        "Found only one user ({}) with {} SSH authorized keys. Would you like to install this user in the system?",
+        user.user,
+        user.num_keys(),
+    );
+    let answer = ask_yes_no(&prompt, true)?;
+    Ok(if answer { vec![&user] } else { vec![] })
+}
+
+fn prompt_user_selection(
+    all_users: &[crate::users::UserKeys],
+) -> Result<Vec<&crate::users::UserKeys>> {
+    let keys: Vec<String> = all_users.iter().map(|x| x.user.clone()).collect();
+
+    // TODO: Handle https://github.com/console-rs/dialoguer/issues/77
+    let selected_user_indices: Vec<usize> = dialoguer::MultiSelect::new()
+        .with_prompt("Select the users you want to install in the system (along with their authorized SSH keys)")
+        .items(&keys)
+        .interact()?;
+
+    Ok(selected_user_indices
+        .iter()
+        // Safe unwrap because we know the index is valid
```

Comment:
I dug because I was curious what happens if `MultiSelect` was passed an empty set, looks like it's this: https://github.com/console-rs/dialoguer/blob/6244c77656e8f3e934a2981c5c7e9dacf938a492/src/prompts/multi_select.rs#L211

OK.